src/controller.cpp

```diff
@@ -1523,7 +1523,7 @@ void Controller::shutdownhushd()
         QDialog d(main);
         Ui_ConnectionDialog connD;
         connD.setupUi(&d);
-        connD.topIcon->setBasePixmap(QIcon(":/icons/res/icon.ico").pixmap(256, 256));
+        connD.topIcon->setPixmap(QIcon(":/icons/res/icon.ico").pixmap(256, 256));
```

Comment:
Why is this change needed?

Reply:
In the **connection.ui** form, I had to change topIcon to a QLabel class for using animations; it didn't show animation without that change. 

By doing so, produced the QLabel error for setBasePixmap. 
`no member named 'setBasePixmap' in QLabel`

After the UI changes were created, the build error was produced via **./build.sh**: 

```
void Controller::shutdownhushd()’:
src/controller.cpp:1526:24: error: ‘class QLabel’ has no member named ‘setBasePixmap’; did you mean ‘setPixmap’?
 1526 |         connD.topIcon->setBasePixmap(QIcon(":/icons/res/icon.ico").pixmap(256, 256));
      |                        ^~~~~~~~~~~~~
      |                        setPixmap
make: *** [Makefile:3645: bin/controller.o] Error 1
make: *** Waiting for unfinished jobs....
```
recommended me to use **setPixmap**.